### PR TITLE
Add basic-illumination

### PR DESCRIPTION
**Pull request by alliecreason**

### CODE REVIEW

File: tools/basic-illumination/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+owner: perssond
```
Comment (bgruening) [inline tools/basic-illumination/.shed.yml:1]: Did you all agreed on one common user for this repo?

File: tools/basic-illumination/basic_illumination.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <description>ImageJ BaSiC shading correction for use with Ashlar</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+ 
+    <expand macro="requirements"/>
+    <expand macro="version_cmd"/>
+
+    <command detect_errors="exit_code"><![CDATA[
+    ln -s $in_files '${in_files.name}' &&
```
Comment (bgruening) [inline tools/basic-illumination/basic_illumination.xml:11]: $in_files needs to be single-quoted

**Review by bgruening — COMMENTED** [review]
Hi @alliecreason,

small review inline. Do you know those tools https://github.com/bgruening/galaxytools/tree/master/tools/image_processing/imagej2

Maybe they are also useful.

File: tools/basic-illumination/.shed.yml (hunk 1 above)
Comment (alliecreason) [reply tools/basic-illumination/.shed.yml:1]: We tried but ran into some issues with changing owner for repos that already exist in the toolshed. So reverting back to the original owner with the lab user have admin access on the repo.

**Pull request merged**